### PR TITLE
dotCMS/core#24163 [Containers] : Clicking up or down arrows in Max Contents field is not working

### LINKED ISSUE #24163: [Containers] : Clicking up or down arrows in Max Contents field is not working
### Problem Statement

When you create a Container and lick the up arrow in the `Max Contents` field, none of the pre-loop, code or post-loop fields show up. They only way to make them show up is by manually typing in a value in such a field.

### Steps to Reproduce

1. Go to the new Containers portlet: http://localhost:8080/dotAdmin/#/containers and create a new Container.
2. Click the up arrow in the `Max Contents` field.
3. The UI never changes, and you only see the `Code` field.

### Acceptance Criteria

The pre-loop, code or post-loop fields must show up as soon as the value of the `Max Contents` field is greater than zero.

### dotCMS Version

Latest master.

### Proposed Objective

Technical User Experience

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

Found during IQA of https://github.com/dotCMS/core/issues/23443

### Assumptions & Initiation Needs

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* show or hide content type on clicking on up or down arrow on max content field

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Comment by zulqarnainvd** [comment]
[screen-capture (10).webm](https://user-images.githubusercontent.com/113915849/220552217-ee654ec6-42bf-4220-b4c6-4fcdf499a93a.webm)

**Comment by zulqarnainvd** [comment]
> 

Old

**Comment by zulqarnainvd** [comment]
[screen-capture (9).webm](https://user-images.githubusercontent.com/113915849/220552835-d8ced5d3-4473-48e7-b4c9-16d5587ef207.webm)

**Comment by zulqarnainvd** [comment]
> 

updated

**Review by oidacra — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts [hunk 1]
```diff
@@ -54,8 +54,7 @@ export class DotContainerPropertiesComponent implements OnInit, AfterViewInit {
                     title: new FormControl(container?.title ?? '', [Validators.required]),
                     friendlyName: new FormControl(container?.friendlyName ?? ''),
                     maxContentlets: new FormControl(container?.maxContentlets ?? 0, {
-                        validators: [Validators.required],
-                        updateOn: 'blur'
+                        validators: [Validators.required]
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts:57]: is there a way to test this?

Comment (fmontes) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts:57]: Is this going to be triggered on every change of the field? is so... we need to catch that somehow.

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts (hunk 1 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts:57]: Yes, Noted

Comment (fmontes) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts:57]: @zulqarnainvd Isn't this updating the UI several time after it goes from 1 to 2 and 3 etc? I mean... we should only change tha state when 0 or 1. Can you double check that?

**Review by fmontes — CHANGES_REQUESTED** [review]

**Pull request merged**